Parse/src/main/java/com/parse/ParseFileController.java

```diff
@@ -41,7 +41,7 @@ public ParseFileController(ParseHttpClient restClient, File cachePath) {
   /* package */ ParseHttpClient awsClient() {
     synchronized (lock) {
       if (awsClient == null) {
-        awsClient = ParsePlugins.get().newHttpClient();
+        awsClient = ParsePlugins.get().awsClient();
```

Comment:
Do we actually need to expose the fileClient builder? It does not need headers, and it just requests public file URLs that shouldn’t require any particular certificate / intercepting behavior.

If we use a single client, all requests to files will send useless (private) headers + any other burden that developer applies to the builder.. right?